.github/workflows/check.yml

```diff
@@ -89,7 +89,7 @@ jobs:
     # https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability
     strategy:
       matrix:
-        msrv: [1.57.0]
+        msrv: [1.70.0]
```

Comment:
Why this large MSRV bump? That's pretty significant (and recent). Is it required by anything we depend on?

Reply:
It is current [clap](https://github.com/clap-rs/clap/blob/master/Cargo.toml)'s msrv.

As far as I understand they have been forced to bump it due to [anstyle](https://github.com/rust-cli/anstyle).

![image](https://github.com/jonhoo/faktory-rs/assets/117771945/cba82c1b-2d9e-48f4-aea5-98f01a2221c1)